src/sdks/core/package.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "@firebolt-js/sdk",
+	"version": "0.10.0-next.10",
+	"description": "The Firebolt JS SDK",
+	"main": "./dist/lib/firebolt.mjs",
+	"types": "./dist/lib/firebolt.d.ts",
+	"type": "module",
+	"scripts": {
+		"validate": "npx firebolt-openrpc validate --input ./dist/firebolt-core-open-rpc.json",
+		"sdk": "npx firebolt-openrpc sdk --input ./dist/firebolt-core-open-rpc.json --template ./src/js --output ./build/javascript/src --static-module Platform",
+		"slice": "npx firebolt-openrpc slice -i ../../../dist/firebolt-open-rpc.json --sdk ./sdk.config.json -o ./dist/firebolt-core-open-rpc.json",
+		"declarations": "npx firebolt-openrpc declarations --input ./dist/firebolt-core-open-rpc.json --output build/javascript/src/firebolt.d.ts",
+		"docs": "npx firebolt-openrpc docs --input ./dist/firebolt-core-open-rpc.json --output build/docs/markdown --as-path",
+		"wiki": "npx firebolt-openrpc docs --input ./dist/firebolt-core-open-rpc.json --output build/docs/markdown",
+		"dist": "npm run clean && npm run slice && npm run sdk && npm run declarations && npm run docs && npm run test && mkdir -p ./dist && cp -R build/javascript/src dist/lib && cp -R build/docs/markdown dist/docs && echo 'Firebolt Manage SDK /dist/ is ready.\n'",
+		"clean": "rm -rf ./build && rm -rf ./dist",
+		"test:setup": "rm -rf test/transpiled-suite && npx tsc --target es6 --moduleResolution node --outDir test/transpiled-suite",
+		"test": "npm run test:setup && NODE_OPTIONS=--experimental-vm-modules npx --config=jest.config.json --detectOpenHandles jest",
+		"prepack": "cd ../../.. && npm run dist"
```

Comment:
what's this about? How come we go back to the root and run this?

Reply:
prepack runs before you do an npm publish. maybe I should change it to prepublish, though.

goal is to avoid pushing any SDK version, even a patch for just that one SDK, without ensuring all of the SDKs still validate, build, and pass all tests.

I'm open to other ideas, but I put this in as an easy first attempt at the above goal.